app/controllers/hosts_controller.rb

```diff
@@ -578,7 +609,7 @@ def action_permission
           'multiple_enable', 'submit_multiple_enable',
           'update_multiple_organization', 'select_multiple_organization',
           'update_multiple_location', 'select_multiple_location',
-          'disassociate', 'update_multiple_disassociate', 'multiple_disassociate'
+          'disassociate', 'update_multiple_disassociate', 'multiple_disassociate', 'rebuild_config', 'submit_rebuild_config'
```

Comment:
These need moving into the "build" block around line 598.  Where they are means that the user will need edit_hosts permission to use the build actions, when the other change (to access_permissions) assumes build_hosts instead.